src/actions/set/set.spec.js

```diff
@@ -0,0 +1,30 @@
+const assert = require('assert');
+const set = require('./set');
+const { SET } = require('../../commands');
+
+describe('set action', function() {
+  it('should correctly return set action given correct input', function() {
+    const events = new Set(['artris', 'lazy-jar']);
+    const parsedCommand = {
```

Comment:
yea so an event name doesn't uniquely identify an event. we need both event name and team id.